app/frontend/components/pages/dashboard/dashboardPage.tsx

```diff
@@ -98,38 +123,35 @@ class DashboardPage extends Component<Props> {
                 name={name}
                 selectedTab={selectedMainTab}
                 selectTab={this.selectMainTab}
+                displayName={name === 'Accounts' && `Account #${selectedAccount}`}
               />
             ))}
           </ul>
         )}
-        <div className="dashboard desktop">
-          {!displayStakingPage ? (
-            <SendingPage shouldShowExportOption={shouldShowExportOption} />
-          ) : (
-            <StakingPage />
-          )}
-        </div>
+        <div className="dashboard desktop">{displayedPages[selectedMainTab]}</div>
 
         <div className="dashboard mobile">
-          {displayStakingPage ? <ShelleyBalances /> : <Balance />}
-          <DashboardMobileContent displayStakingPage={displayStakingPage} />
-          {!displayStakingPage && shouldShowExportOption && <ExportCard />}
+          {displayedSubPages[selectedMainTab]}
+          <DashboardMobileContent selectedMainTab={selectedMainTab} />
+          {selectedMainTab === 'Sending' && shouldShowExportOption && <ExportCard />}
```

Comment:
'Sending' should be a constant, otherwise if we rename the tab, it may easily break, if I understand correctly

Reply:
I agree I have some ideas about refactoring the dashboard page since it has too many constants and is cumbersome to work with, Would you mind if I made it into another PR tho?

Reply:
no, as the usage of hardcoded strings seems widespread, it seems sensible to deal with this separately - worth creating an issue though so we don't forget about it even if priorities shift after merging this